tests/src/test/java/com/netflix/msl/msg/MessageHeaderTest.java

```diff
@@ -599,15 +599,15 @@ public void entityAuthDataPeerMslObject() throws MslEncodingException, MslCrypto
         assertEquals(RENEWABLE, headerdata.getBoolean(KEY_RENEWABLE));
         assertEquals(HANDSHAKE, headerdata.getBoolean(KEY_HANDSHAKE));
         assertTrue(MslEncoderUtils.equalObjects(MslTestUtils.toMslObject(encoder, CAPABILITIES), headerdata.getMslObject(KEY_CAPABILITIES, encoder)));
-        assertTrue(MslEncoderUtils.equalArrays(MslEncoderUtils.createArray(p2pCtx, PEER_KEY_REQUEST_DATA), headerdata.getMslArray(KEY_KEY_REQUEST_DATA)));
+        assertTrue(MslEncoderUtils.equalArrays(MslEncoderUtils.createArray(p2pCtx, ENCODER_FORMAT, PEER_KEY_REQUEST_DATA), headerdata.getMslArray(KEY_KEY_REQUEST_DATA)));
```

Comment:
As changed, this test will fail when two or more encoder formats are supported. Better to compute format as
final MslEncoderFormat format = encoder.getPreferredFormat(CAPABILITIES.getEncoderFormats());
and use that in place of ENCODER_FORMAT in the three lines changed here.